### PR TITLE
test: make tests for aws run parallel

### PR DESCRIPTION
**Pull request by jgrumboe**
On my local Mac, `make test_aws` is now ending in under 7 mínutes.

```
PASS
coverage: 92.8% of statements
ok  	github.com/infracost/infracost/internal/providers/terraform/aws	389.751s	coverage: 92.8% of statements
```

### CODE REVIEW

**Comment by jgrumboe** [comment]
Number of parallel tests will depend on `GOMAXPROCS` or `-parallel N` argument.

**Comment by jgrumboe** [comment]
First tests without tweaking parallel numbers were running for 14 minutes. 

Let's throw in `-parallel 10`.

**Comment by jgrumboe** [comment]
10 parallel didn't change anything it seems.
Maybe try 15 just for tests.

**Comment by jgrumboe** [comment]
Last test run will be decreasing parallel tests to 2.
Increasing it didn't have any benefit.

**Comment by jgrumboe** [comment]
@alikhajeh1 It seems that a parallelism of 2 is the best choice. This will bring down AWS test from ~20min to ~13min.

**Comment by jgrumboe** [comment]
If we fix `-parallel N` to  a certain value within the Makefile, this will influence local development. With last commit we let go decide what's the best value for parallelism.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,22 +4,31 @@
...
-	- [Test](#test)
-- [Adding new resources](#adding-new-resources)
-	- [Glossary](#glossary)
-	- [Quickstart](#quickstart)
-	- [Cost component names and units](#cost-component-names-and-units)
-	- [Finding prices](#finding-prices)
-	- [Adding usage-based cost components](#adding-usage-based-cost-components)
-	- [General guidelines](#general-guidelines)
+- [Contributing to Infracost](#contributing-to-infracost)
+	- [Table of contents](#table-of-contents)
```
Comment (aliscott) [inline CONTRIBUTING.md:8]: Not sure we need a link to this since we are already in the Table of contents here

**Review by aliscott — APPROVED** [review]
Thanks @jgrumboe this is a great improvement.

**Pull request merged**